osidb/filters.py

```diff
@@ -248,6 +250,25 @@ def exclude_fields_filter(self, queryset, name, value):
         return queryset.defer(*list(valid_fields))
 
 
+class FlawQLSchema(DjangoQLSchema):
+    """
+    Limit the fields that can be queried in the DjangoQL query.
+
+    This is a subclass of DjangoQLSchema that limits the fields that can be
+    queried in the DjangoQL query to the fields that are allowed in the
+    FlawFilter. This is necessary because the DjangoQLSchema allows querying
+    any field in the model, which is not desirable in this case.
+    """
+
+    def get_fields(self, model):
+        fields = super(FlawQLSchema, self).get_fields(model)
+        if model == Flaw:
+            exclude = ["acl_read", "acl_write", "snippets", "local_updated_dt"]
+            fields = set(fields) - set(exclude)
```

Comment:
Is nested query supported? (eg. `affects.impact`) Is so, these nested models also do have the acls.

Reply:
changed to always exclude `acl_read, acl_write`